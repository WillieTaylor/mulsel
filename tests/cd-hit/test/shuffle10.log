================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.90.out -c
         0.9

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        364  clusters

    10021  finished        365  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.80.out -c
         0.8

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        187  clusters

    10021  finished        187  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.27
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.70.out -c
         0.7

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        120  clusters

    10021  finished        120  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.28
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.60.out -c
         0.6 -n 4

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 3M = 3M
Miscellaneous   : 0M
Total           : 15M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98011432

comparing sequences from          0  to      10021
....................    10000  finished         84  clusters

    10021  finished         84  clusters

Apprixmated maximum memory consumption: 15M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.50.out -c
         0.5 -n 3

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98381872

comparing sequences from          0  to      10021
....................    10000  finished         63  clusters

    10021  finished         63  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.40.out -c
         0.4 -n 2

Started: Fri Dec 11 18:04:29 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98399512

comparing sequences from          0  to      10021
....................    10000  finished         37  clusters

    10021  finished         37  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.24
