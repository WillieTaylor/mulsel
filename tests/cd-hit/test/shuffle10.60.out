>seqI424 = I family seq 424  0.002790 34 I
NIWGVCIKMITYGIIIIIIFPVIIKIIVIIIITPISIEIPIKVIIQTMIISINQVVILIMIPRISIIGIIVYE
>seqN47 = N family seq 47  0.005802 72 N
YNNNNNNNNNNNNNNNNNNNNNNNINNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
>seqI519 = I family seq 519  0.005806 34 I
INIQIVIIIVIEMTYIIIIPFPMVFKIIEKVIIIQTVIVEENTIIKPNIIILIIINCVVDILIRMICIIISIIGIRDVE
>seqI279 = I family seq 279  0.007653 40 I
NCVVIRMMIIEIIINIRILISGAVWIIIIVMIIEPIIIIIIWIIIIIMIIIINQIICILIMINIIRIIIGFR
>seqT178 = T family seq 178  0.009715 66 T
PAICTTTTTLTTTTTTTTTTTITTTYTTWTTTVTTTTRTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTNTTTTTYTTTV
>seqC58 = C family seq 58  0.011272 70 C
CCCCCCCCCCCCCCCCCCCCCCCCCCCCRCCCCCCCCCCCCCCDCCCCTCCCCCCCCCCCCCCCCCCCCCCCC
>seqK41 = K family seq 41  0.013765 77 K
FKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKKK
>seqM261 = M family seq 261  0.015076 50 M
MKPMMRPMMMMMMSCMMMEMMVMMMWWMMTSMMMMMMMMMTWQMMMCMWSPMMIMMMMMMMVMMMRMMMMMARMHQPDD
>seqY598 = Y family seq 598  0.017382 34 Y
YMERLELYYYYYYYLQAYYYYKSYYYGEQIYDYYLETAYYEYDTHHIYEETYQHHYGQGYYTYEAYYGEMRYEYYYYN
>seqE596 = E family seq 596  0.018960 36 E
DKEEEHHYDCWENGEGTEVEEAEEESEENNEREEEWEPQKDEECEETEWEHEVAETTKVYEENEEIEIEIQEELEEAKYW
>seqE265 = E family seq 265  0.018974 44 E
KCGHLEEDEEEEFEGECEENVDMEEEWEEEQEEEEEKIPEEFEEEQLTVEEEEEEEECEWETLVENEWEEEFEYEELQFK
>seqA28 = A family seq 28  0.019433 74 A
RAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
>seqE496 = E family seq 496  0.021080 34 E
FEEEHHEEEEEPEEEGTIVIIHAQESTFEEVEYEEEEEPEKIECCTKEVEHIVPEQTEVDEENDQIEEIELEEAKE
>seqY213 = Y family seq 213  0.021769 54 Y
APNCYALYYYYYYYYTYYYYPYYPYYLYYYFYYYWYYYYYYYYYVYTYYYYYYYYYYYYYYVWYYYYYIYLPYALWKW
>seqV466 = V family seq 466  0.029478 36 V
GVVRVWVNVLCVENVEHSNVVVTVVVLGAVRVVRQFVHPDVTVVVVVVWPKDVVFVNVVVVVSVESRNGAVVV
>seqL587 = L family seq 587  0.030079 34 L
LQIIALDLLLLHELLTRCLMCWQVLLLELLCERLHQLTAQWLLKLLYLLLLQHLLLGFLLLYPLQLYHLLTW
>seqV24 = V family seq 24  0.032825 75 V
VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
>seqH387 = H family seq 387  0.038074 34 H
PTMWHPHKHHTLHHKHHAMIHHHHWHDIHPHHRWKHHHHHFHHHHATAHHGRDWHHKHNAHLHHMRKHK
>seqS169 = S family seq 169  0.038930 64 S
FNSESSSSSSSSSSSSSSSLSSSSSSSSSSSSSWASSKSSSSSSSSSSQSSSSLSSSSNSSSSSSSTESSSSSSSSVN
>seqE379 = E family seq 379  0.050080 34 E
KIEHEEDFEPEEYEGTEEEIHAQESWQEEVEEEEEKIPEEFEEIATVQDEEEHIEPEYTEESEKNTEEYELQEANKI
>seqH592 = H family seq 592  0.054113 34 H
HPRHDWHHCPLHHRHHHHQHHCHHCHGVTHHHHHDIEPHTHCHNECDHHHNYDFAHIYHHPGYMHHHNAFHYLTHDT
>seqP472 = P family seq 472  0.054910 35 P
NLNPMFLKNPPPQPPQPPFPQFNLNWLGPSQPWGLLPYNQHTPPVHPFLPPLPQPPPTEPPPPPWPPPKPSKPPPPP
>seqR370 = R family seq 370  0.056344 35 R
IYFTDMPLRRPSTRMRARRRRDYIERKRVLRKDRRTTPGRRRIRVTIDRKRRRMQRRRRMVNRRYERRRMRRKVGLRPRQR
>seqL365 = L family seq 365  0.058535 36 L
MTLLEFWLLAELLFEELCLLVNWLMCGYLLLLQLNMLEALSLLIQDLPLLLSKLLLMHLHLCLLQLHYGLLYLLMQLQ
>seqQ185 = Q family seq 185  0.061187 56 Q
RQQSGQQVQQQQQVQHSQMQQQQAQCYQQQQQQQQQMQQQQRQQRQQQQQQQQDSQVEQQQQQQQQSQQQQQEQQQPT
>seqF390 = F family seq 390  0.062097 34 F
QFPKWFFFFFFVFDFKYFYFFFFFMFLTFQLAFIQLRSNFGFMLDFFDWAWFYFTFRSFFFLFFVFKYYTIMQFFLFPQRW
>seqM393 = M family seq 393  0.064986 34 M
YTWQDGGMRMIMMMMFMMMEMMMMIWWAMTSFMMMMMMIAWPVMMNMWSLAWIMRFMMMMMMMMMNMVARHYHQTPN
>seqI169 = I family seq 169  0.066892 64 I
CTIIIIIIIIIIIIIILIIIIIIIIIEIIIIIPIIIIIIIIWIIIIIIIIIIIIIIIIILIIIMIIIIIIIIGAF
>seqC359 = C family seq 359  0.067600 36 C
PGCCCACCARKCNLCACCRCFMRCCHCCACCCNMSCDCCCCCPCDCNCSCCCSCWWCNCNCAANNCLQCCTYLR
>seqM594 = M family seq 594  0.071724 34 M
QMLMMCQHLDYMMMMWHMMKPRMEPELMWPMISFMMMMKMRMMMNMLLAKVHVMMMMMAWCHMMMMQMMMMPGSN
>seqS488 = S family seq 488  0.072931 34 S
SSRSNSVESLSSGFNSEVGSSMSSSVNSITQQSMSSSSSSILIGSKSVPLSSYQYGSDMGGGSSLSSQVSSSYISVN
>seqA571 = A family seq 571  0.074108 36 A
FARYYAYKAYAALNNAAAAAGACLAAAAARAAFQAAEYAAIICKVAIDRAVAYACKLAQAAYSEAQNSALGRSLANLCAAAGA
>seqA492 = A family seq 492  0.088307 34 A
FARHAKYAALANALAASGAALAAASAARAAFAAAEAKEVCMAARYKAYACKLHQALAAMQEAAAGRSSYLTARIGAA
>seqI594 = I family seq 594  0.095723 34 I
WIIIWKGIKIVICEMYIIIIIIFRIVILIEVIIMIIVNIEENTNIKPLIIIFILNAVVDILGIRFICIIIIIKIRDVE
>seqK525 = K family seq 525  0.097475 34 K
MIKKKKGKKKVKKPKKVKKKKKKEKKYQTVLSKKKNMYTHMCSANHKKSSDEKTDTIDKVNKKCKKKKNKCK
>seqW540 = W family seq 540  0.099026 35 W
WWKWWWPQWWWWNWWFFWWSTQFLWWWLWPNNACPWWFRWKVMFWWWWPPQCWWFVWWNWDFENCWWGEWWGWFIWPM
>seqL198 = L family seq 198  0.106961 60 L
MTMTWLLLLLLLLLLLLLLLLLLLLCLGLLLLLLLLLLLELLLLLIWLLLPLLVLLKLLLDLLLLCMLKLLLLLYLLLKW
>seqV372 = V family seq 372  0.107828 34 V
GMVKVWWNVLVCVVVVSTVVVTTVVVLWAVDVVNVTVHVHTVVGYVLWCKDVVVFQNVVVFRVVVVSRNGAVW
>seqW163 = W family seq 163  0.115366 64 W
WWWWWNWWWEWWAWWIWWWVWWWWWWWHWWWWAWWWWWWWWWWWLWWWWWWWSWWWWWWWWWWWPWAWWWWWWWWH
>seqR597 = R family seq 597  0.117120 35 R
VCRRFDFSRRMCYRRRRNTFVQVIRRHRARKDRMFAWGARYRWRRRRRRRAKVRRQRFATRHRVEFTRRRFRRTVRRTFPRQK
>seqD451 = D family seq 451  0.133209 34 D
DDLDSDASDFCDLDMANDDDDPWDQDEDVDRRADCLDFSDDDRIHDDKDYQASADDDWQIMDIDMDIRQDSDRDDD
>seqP241 = P family seq 241  0.139677 46 P
DDYPPPPLPWPPPPPPPDFPPPPPRPKPPPPPPPPFPYPPPQPTMYFPPPLPKQFPPPTEPPPPYIPPPPQL
>seqG377 = G family seq 377  0.154756 35 G
FYGNGGGGGGGNWGGHTGAPGGHGGEWGWSQQGCPGGRSPGGGGGTKGGCMNNGGLRWGGRGGLAGPELKQGPFQPFQG
>seqL488 = L family seq 488  0.158303 37 L
ELTLEFWLLLLDELLLLELLDVDMCGYLVLLLYLERLSLLLAQCDLLLLSGLLSHLLLCIYLLYGKYLLLLLP
>seqA256 = A family seq 256  0.163760 41 A
SYKYANKAVHAAAAATAYLAAHAAKAAVATAAAFAAAVAYAYAAIAAAAADNAAAAAAECAVCAAAAQYYFNN
>seqR454 = R family seq 454  0.216649 36 R
QVRFTDMRRRRPIRRMRAECRRDIERRARKDRRRAPGARRIRVTRHDHRRKRRRQRARVRMVMYNRQARERDLRGRRPRQR
>seqD573 = D family seq 573  0.222011 36 D
MDDDDMSASDDTDDIEDLRCDDIQMQESVDDEDTTCLDTSQDDDDIDDDQYDAPADQDEDMDHDMDKIDDQDDIDDD
>seqS577 = S family seq 577  0.233759 34 S
SSRFSYSSSESNSFGFNRFGNSTLMSVSSITSKSMSGSSFSSSSKSTVSMYSKDMGGGSSLSSLSSTASISSSN
>seqT391 = T family seq 391  0.236609 34 T
TKCITFLTTTTTLDTTTGTTITPPMTLWTTTGYNTRTIHTTNKYTTLTSWTTTKTITTTEILTTNMKTNAYHPHGVMFIG
>seqN281 = N family seq 281  0.237917 38 N
CYEAENNQLNPNNNNNNNNENCNIQRNGELNNNPENNGSQNNNNNNESWCNNVENNNNNNSVNNNQNQNPSPL
>seqG197 = G family seq 197  0.242544 52 G
YNSGGGRWGGGGGGGGHGSGGGGLLKGGGGGGGGGGGGGGGGSGGGGMGPGGGGWGGGQGLGGGPGKGGIPFGYL
>seqF167 = F family seq 167  0.247961 64 F
RDFFFFFFFFFFFFLFFFFFWFFFFFFFFFFLFNFFFFFMFFFFSFFFKFFFFFFFFFFFFFFFFDFFFFFFFFYC
>seqN584 = N family seq 584  0.254011 34 N
NHNNHRNDNCNWQNPNNRNPNNIVNTHANQHWHMLNNNNANNSNINHEENNNNRSNVPDWNTNANFPEYNTNWSNFN
>seqR258 = R family seq 258  0.272780 53 R
IYFIRPPRRRPRRRRRRRRRDRIRKRRRSRRRRFRRRRRRRRMRRRRRRRRSQRRRRRRRRCRYVRYRRRRRKVGVQ
>seqK445 = K family seq 445  0.276123 34 K
MIKKKRKGRKKKVIKKPKMKKKIKVKIEKKDTLSSKKIKMKLKGHMCKKKNHRKPSIDEKTKIKNVNKKKKKDVNECM
>seqM486 = M family seq 486  0.312235 34 M
QMYMCMQQDYEMMWMMWMMNKIGRSMPHMMWWAIISFMMMMMMHWSPVMMNMMSLARKDHWMMMMSMMMCYMMMQRMWEHQTNM
>seqP367 = P family seq 367  0.319171 35 P
RYMLKPPPPQLPTPPPPPQFNPLPRPKIWPPQPPHGPLPPQHTPYPHPFLQFGPPTEIPPPPFIPPSPPPSMHP
>seqP574 = P family seq 574  0.347508 34 P
NPNPPPGNPPPTPPPLFPFPPIPCLPWPMHQWGLPNQCTPPVPPFPLPPPPQPFMTPPPPKWPHCSLMPMP
>seqK319 = K family seq 319  0.351780 34 K
LIHPKKKRIYKKTKPKAKKSNDKQKKKETYSKKKIKKKKKMKKKHHKKSKTKDKKRIKVNKKIWKDCAERD
>seqF523 = F family seq 523  0.354857 34 F
DFFFFWFIFFFFDFFFFFGFFRFMQFAFFFRLFFFFCDFTWFIWYFQPFFNFLFSFKQETMQPLLFPQRWQ
>seqH237 = H family seq 237  0.371672 54 H
MCVHHHHHMHHHHHHHHHHHHHHHHHHHVHHHHPHHHHRHKHHHHHHFHHHHHNTHHHHGFHHHKHHPHRMRK
>seqV599 = V family seq 599  0.379219 35 V
NYVMVSQTHQCEVVVHCNVVVTVHVVLGVSVPVLVVVVVVCHQTVRCFVDVVNVVVPVVSVLSSVHSGVVWVV
>seqT505 = T family seq 505  0.396160 34 T
MTMEFTFTLTTRTSDTAESGTTTTDPTCWTTDTYTTMCHTTNSCTTLMFTTTTKTPHTTHELTMTNTKHRTYTYCTMHFKIG
>seqH475 = H family seq 475  0.416910 35 H
HPHRHWNHWHLHKHHHLQHWTHCAHHIHHHHHWHDIWPHTHHACHHHFHIAHAAMGHHHNGRAMHHHNAHLTIRTKAHK
>seqQ504 = Q family seq 504  0.422949 34 Q
LQGQRQQKVVQQQELQQFMFFQGDRQRYQQCQQDQNKQWQGQKEQKMWDVQQQFQADGQQTQQQCQQRQQDRTLQPGK
>seqY461 = Y family seq 461  0.440304 35 Y
YMYYGRYFKYDYTYYSYYYYYVNISDYYYYTYYYYDVYIYYYYHHHYYYYYCMGYYYLMFYHWELN
>seqV232 = V family seq 232  0.485306 43 V
WWNVVLVHVVDVVVHVVVPDVTWVVVLVVAVVDVVVNVVVVHVHTVVYVVWCVDVWVLQNVVVFVVWVVVIVN
>seqG596 = G family seq 596  0.488609 34 G
GEGYGGGGGGTGLGYWPEGGSHPGGKPGSSHGGGGGCGKNKPTGSLFLGGGHGPGHGGGGGYGEFG
>seqG522 = G family seq 522  0.512779 34 G
GGGGGGGQTHTGALGGGDKAEWSSHLPGGPGYSHGGGGKGGSNGDGPGLRGPGGHGGPNHGGGPFGYGFG
>seqD131 = D family seq 131  0.544802 67 D
NIWDDDDDDDDDDDDDDDDDDIDDDWDDDDDDDDDDDDCDDDDDTDDDDDDCDDDDDDDDDDDDDDDDDDDDDDDY
>seqW360 = W family seq 360  0.559319 35 W
WPPWLPWCHEEWWWQLVGWWITWWGWWKLHWRNAFWWRWWRWKWWWIPPWRCIWWKWWMWDVPWWCPWWWWHVGWWIR
>seqN403 = N family seq 403  0.633863 34 N
NINNVNNNCNLNNRFNNNPNNNNENCNQRWHQCNENPIVNNNNSQNIWQCELEMNNVRNNNNNSMMISQNINPHSCRHM
>seqQ353 = Q family seq 353  0.642107 34 Q
QHVGQQQVTSQELHHVYMQQQVDAFCRYQQSQLQQQNQASARQQQKQKMQDSQLEQMQADFQHQQSQQQQQEQHRTQQPL
>seqF593 = F family seq 593  0.647475 35 F
DFQHFFFVILFFTVFFNVFFNQFFMTFQFPFFFLNFFFHCFFIFVIWYFQFFMFLFQFKFTEVFMQFLLFPCQFWQF
>seqA426 = A family seq 426  0.665350 34 A
AQSAKYANKAAAAAAAAVAFLAACSAQAAAVGAAAAASVCMAANYMAACIKQGIAAAMAECAVAGRSSMSTAVIGMT
>seqD328 = D family seq 328  0.691511 36 D
SIMCSDDDDDCDDDINDIDTWDWEDSDDRDNDCSDDDSYKDTIHDKDDQWDADRHWHIMDIDDDDIDDDDDKED
>seqS394 = S family seq 394  0.739591 34 S
FRSNSVVELSIGSNSSSSSSMMSSLSFRDSWQSRMSSKFILKGSKSSSLSSYQYSGSLDMIGNAGSSLGSSSVSCASSCYASVN
>seqN507 = N family seq 507  0.745134 35 N
NHNNHRNNNCNDQNPNNNYPNNIETNANQNWHQHNENIVANNNNSQNIGFQENEMNNRHNVFNDNNSNNFASYNILSNMN
>seqY534 = Y family seq 534  0.753416 36 Y
YMYLGKYYYYFHYDYLRYFYYSSYYYYAIYDYYYYYTQYYYYDTVYIETYYHHYIYQYTYGGYYGELMRYEHYELN
>seqW464 = W family seq 464  0.785193 35 W
WPPWQWCWWNWGVWGMIWWWCWWWWLHWNAWWWWRWKWEWWEPPQCWWWWWWWNWDFWNCWKWEVGWFIM
>seqT578 = T family seq 578  0.833470 34 T
MTQEFTTTLCTRTSDTTETRGTTTDPTCWTTTQYRTTCHCTNRCTTAPFENPFTTETLTTTTETRQNTSTMHCDTTT
>seqV537 = V family seq 537  0.875100 35 V
GYVEVWVVVTLECCVEVVVHCNVVVTIVVLGYRPVVQFVVVNVTVVQVVRPFVDVVQTPVVVVSSRNGAVVVV
>seqC545 = C family seq 545  0.941423 38 C
CKYCACCCGPCICCLTCCCCCCCCRCCCCCPVCTNFCFCCVCDAHCCCCMNPKYYQCAHCQCCCCCLC
>seqK590 = K family seq 590  0.951817 34 K
MIKFKYKVWKGVFKKVPKKKEVVKKKKKKEKYKKVPKSKKDKNMKTNHVWQLSNHQKKSDQDTDTKQVNNDKCKKKKINYKKK
